### ISSUE TITLE
User role:code in each view that presents code or filenames

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#97014]
Currently we set `role:code` it on the editor, we need to also set it:
- [ ] On every tree and list except Extensions view. Since there is an exception, the list/tree client would need to be able to opt out of this
- [ ] on every input box (like the find widget and search input)

fyi @joaomoreno
For now assigning to On Deck until we get more feedback. Also I think currently Orca screen reader is the only one respectin the `role: code`.

**Comment by isidorn** [comment]
Closing this as out of scope and we can jump on this in the future
@joanmarie let me know if you are passionate about this and I can reopen

**Comment by joanmarie** [comment]
@isidorn I'm not passionate about this. By the same token, if Orca users complain that Orca isn't presenting code or filenames correctly and they live in containers which lack the `code` role, the fix IMHO will be to use the `code` role in that view; not for me to try to add some heuristic in Orca.

Long way of saying, maybe the best thing to do is handle each complaint (if any) on a case-by-case basis.

**Comment by isidorn** [comment]
@joanmarie makes sense to me. Thanks a lot!
